src/stats/probs/cdf.rs

```diff
@@ -199,6 +206,12 @@ impl<T: Ord> CDF<T> {
 
     /// Downsample CDF to n entries. Panics if n <= 1 and returns identity if n is greater
     /// than the number of entries.
+    ///
+    /// Runtime complexity: O(x), where x is the original number of `Entry`s in `CDF`.
```

Comment:
```suggestion
    /// Runtime complexity: O(m), where m is the original number of `Entry`s in `CDF`.
```

Reply:
OK, if `m` is more canonical than `x`... :man_shrugging: